src/components/Card.js

```diff
@@ -27,6 +51,22 @@ export class Card {
     this._cardElement.querySelector(
       ".card__image"
     ).alt = `Photo of ${this._name}`;
+    this._cardElement.querySelector(".card__likes-counter").textContent =
+      this._cardLikes;
+    // this._owner == !"Angel Baez" ? this._deleteButton.remove() : null;
+    console.log(this._cardLikesArray);
+    this._cardLikesArray.forEach((element) => {
+      // console.log(element);
+      if (element._id == "b467d8eeb238431481a2ba2b") {
```

Comment:
`_id` of the user should be received from the server only 1 time. You need to store it in a variable `userId` or in class `UserInfo` and use it when you create a card passing it to the constructor of `Card`